### ISSUE TITLE
ios Only : screen loses content while scrolling on a particular screen

### ISSUE THREAD

**Issue by aditjain** [appium/java-client#431]
screen loses content when automated test starts to scroll (using either 'auto-scroll' or 'scrollTo' features). 
I do not face this issue when i run the app manually only after using automation I face this issue.

I want to provide a video for the issue but do not see any option here. 

below link has issue captured in more detail.. 
[https://discuss.appium.io/t/ios-only-phone-screen-loses-content-when-automated-test-starts-to-scroll/11017](url)

**Comment by SrinivasanTarget** [comment]
Above methods are deprecated and we suggest to use latest version of appium i.e 1.5.3 now and java-client 4.0. Refer these for scrolling: https://github.com/appium/java-client/blob/master/src/test/java/io/appium/java_client/ios/IOSScrollingSearchingTest.java & https://github.com/appium/java-client/blob/master/src/test/java/io/appium/java_client/android/AndroidSearchingTest.java

**Comment by aditjain** [comment]
Hey Srinivasan, thanks for replying but my issue is not related with scrolling. I just mentioned the scrolling method for extra information. 

I would really like to send a video of the issue that i am facing since its hard to explain. if you can provide me any email or something that would be great. 

this is a ios only issue and only happens on a certain screen while scrolling on that particular screen. On other screens there is no problem with scrolling.

**Comment by SrinivasanTarget** [comment]
@aditjain We are ready to hear and sort issues but if the issue template is followed properly that would give a clear picture to everyone.I once again suggest you to use the latest version of appium client and server and log a new issue here https://github.com/appium/appium/issues following the issue template incase if you hit the same.Also regarding video please upload on any cloud (google drive,etc) and give us access to it look further.
